Thumbprint/Thumbprint.xcodeproj/project.pbxproj

```diff
@@ -613,6 +614,7 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				SWIFT_COMPILATION_MODE = wholemodule;
+				SWIFT_STRICT_CONCURRENCY = complete;
```

Comment:
This is incorrect, you need to put it in quotes.  See here: https://github.com/thumbtack/ios/blob/ee6d008a9d9f271bb36197c37dfaa51e760af392/Xcconfig/Sources/Generator/main.swift#L34

Reply:
it's not in the generator file. I has been modified by xcode. see the code above:
`SWIFT_COMPILATION_MODE = wholemodule;`

Reply:
I see. However, i think this does nothing as these are on the Playground App & the TestHostApp, they aren't on the framework as far as I can tell. 

I'm a bit confused about this xcodeproject in general. I can't get it to build for me, it says it's missing the thumbprint package.

Reply:
Looks like to add it to the framework, you need to add it to the package file: https://www.swift.org/documentation/concurrency/ 

```
.target(
  name: "MyTarget",
  swiftSettings: [
    .enableExperimentalFeature("StrictConcurrency")
  ]
)
```

Reply:
I will make the framework StrictConcurrency in a follow up PR. I have made PartialSheetPresentation MainActor